### ISSUE TITLE
iron-overlay-behavior accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-overlay-behavior#11]
Focus: focus state on this page is not very visible and should be revisited. 

In general, this appears to be working fine with screen reader and keyboard navigation (testing just with Chrome / Mac / VoiceOver today as I don't have corp access on my Chromebook or Win machine remotely). Upon opening one of the overlays, I am able to navigate to the text / buttons within the overlay, but I can also navigate to the other options at the top of the page while the overlay is still open. 

These overlays would seem more intuitive for screen reader users if focus stayed within them until the user took an action, e.g. activated one of the buttons or pressed escape. I don't think it should be possible to navigate to any of the background content before first closing out the dialogue. What are your thoughts? Is that not your intent here? 

Also, we should consider making the titles of the overlays speak as announcements once open, so the user has better context about what just opened.

**Comment by morethanreal** [comment]
Won't fix because this element doesn't implement UI. The demo element is only to show behavior.
